example/Jamfile

```diff
@@ -44,11 +52,13 @@ local targets ;
 
 for local s in $(sources)
 {
-    targets +=
-        [ compile $(s) :
-            [ ac.check-library /libjpeg//libjpeg : <library>/libjpeg//libjpeg : <build>no ]
-        ]
-        ;
+targets +=
+    	[ exe [ regex.replace $(s)  ".cpp" "" ] :
+	    $(s)
+	    /libjpeg//libjpeg
+    	    /libpng//libpng
+	]
+	;
```

Comment:
I'd suggest to keep some level (typically, 4 spaces) of indentation of body of the `for` loop.